src/lib/utils/thorchain/constants.ts

```diff
@@ -9,3 +9,6 @@ export const THORCHAIN_POOL_MODULE_ADDRESS = 'thor1g98cy3n9mmjrpn0sxmn63lztelera
 export const THORCHAIN_BLOCK_TIME_SECONDS = '6.1'
 export const thorchainBlockTimeMs = bn(THORCHAIN_BLOCK_TIME_SECONDS).times(1000).toNumber()
 export const RUNEPOOL_DEPOSIT_MEMO = 'POOL+'
+
+// Number of blocks to withdraw form RUNEPool (https://gitlab.com/thorchain/thornode/-/blob/develop/constants/constants_v1.go#L117)
```

Comment:
```suggestion
// Number of blocks to withdraw from RUNEPool (https://gitlab.com/thorchain/thornode/-/blob/develop/constants/constants_v1.go#L117)
```

and potentially more informative:

```suggestion
// Number of blocks from last deposit before user can withdraw from RUNEPool (https://gitlab.com/thorchain/thornode/-/blob/develop/constants/constants_v1.go#L117)
```